crates/pipeline_manager/src/db/connector.rs

```diff
@@ -103,12 +103,14 @@ pub(crate) async fn get_connector_by_name(
 
         Ok(ConnectorDescr {
             connector_id,
-            name,
+            name: name.to_string(),
             description,
             config,
         })
     } else {
-        Err(DBError::UnknownName { name })
+        Err(DBError::UnknownName {
```

Comment:
(Not this PR, sidenote: we should add `DBError::Unknown{Pipeline,Program,Connector,Service}Name` in the future as they are different namespaces)

Reply:
Agreed. Not a fan, especially given I already added a `UnknownProgramName`.